.github/workflows/dev-image-update.yaml

```diff
@@ -57,6 +57,11 @@ jobs:
         uses: "broadinstitute/github-action-get-previous-tag@master"
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Build nmp build for new ui container
+        run: |
+          git pull
+          GCR_TAG=$(git rev-parse --short HEAD)
```

Comment:
Can you move this into `Pull down new tags`? I don't think `git pull` or `$GITHUB_ENV` are still needed, so the only thing left from this task is the `git rev-parse`, which you could do in that task instead.

Reply:
Hmm, apparently `GITHUB_ENV` is a github actions thing, so maybe it's still needed. Pretty sure the `git pull` can go though. Either way this is no longer a `build` task.